spec/services/cfe/create_applicant_service_spec.rb

```diff
@@ -1,30 +1,79 @@
 require 'rails_helper'
 
-module CFE # rubocop:disable Metrics/ModuleLength
+module CFE
   RSpec.describe CreateApplicantService do
-    # uncomment this if you need to connect to a real instance of Check Financial Eligibility service
-    # running on localhost
-    #
-    # before(:all) do
-    #   VCR.configure { |c| c.ignore_hosts 'localhost' }
-    # end
-    #
-    let(:application) { create :legal_aid_application, application_ref: 'L-XYZ-999' }
-    let!(:applicant) { create :applicant, legal_aid_application: application, date_of_birth: Date.new(1999, 9, 11) }
+    let(:application) { create :legal_aid_application }
+    let!(:applicant) { create :applicant, legal_aid_application: application }
     let(:submission) { create :cfe_submission, aasm_state: 'assessment_created', legal_aid_application: application }
-    let(:faraday_connection) { double Faraday }
-    let(:connection_param) { double.as_null_object }
-    let(:expected_payload) do
+    let(:expected_response) { expected_response_hash.to_json }
+    let(:service) { described_class.new(submission) }
+
+    describe '#cfe_url' do
+      it 'contains the submission assessment id' do
+        expect(service.cfe_url)
+          .to eq "#{Rails.configuration.x.check_finanical_eligibility_host}/assessments/#{submission.assessment_id}/applicant"
+      end
+    end
+
+    describe '#request payload' do
+      it 'creates the expected payload from the values in the applicant' do
+        expect(service.request_body).to eq expected_payload_hash.to_json
+      end
+    end
+
+    describe '.call' do
+      context 'response received from CFE' do
+        before do
+          allow(application).to receive(:applicant_receives_benefit?).and_return(true)
+          allow(application).to receive(:calculation_date).and_return(Date.today)
+        end
+
+        describe 'successful post' do
+          before do
+            stub_request(:post, service.cfe_url)
```

Comment:
I'd recommend turning VCR off around the stub request call - then any problems raise WebMock errors rather than VCR errors. Something like this:
```ruby
    around do |example|
      VCR.turn_off!
      stub_request(:post, subject.cfe_url)
        .with(body: subject.request_body)
        .to_return(body: api_response)

      example.run
      VCR.turn_on!
    end
```
That is, if you change `subject.cfe_url` to `http://foo.com` and run the test, if VCR is on you'll just get a message along the lines that there isn't a VCR cassette for that call. If VCR is off, WebMock will return a much more meaningful error message, including an example of a `stub_request` configuration that would pass - which you can then compare with the one you have.

Reply:
Turns out that there is a peculiarity with with `around` as used in your example above - it doesn't instantiate the variables defined with a `let!` first.  Solved by putting the `around` at the group level, but worth bearing in mind if you get similar problems in the future.